src/modlistview.cpp

```diff
@@ -1232,6 +1355,73 @@ void ModListView::timerEvent(QTimerEvent* event)
   }
 }
 
+void ModListView::mousePressEvent(QMouseEvent* event)

```

Comment:
Can you add a short comment as to what this and `mouseReleaseEvent()` are actually doing? Something like "alt+click selects all mods inside a separator if the option is enabled".